src/tests/topp/FileMerger_6_output.mzML

```diff
@@ -199,9 +199,6 @@
 				</scanList>
 				<precursorList count="1">
 					<precursor>
-						<isolationWindow>
-							<cvParam cvRef="MS" accession="MS:1000827" name="isolation window target m/z" value="445.34" unitAccession="MS:1000040" unitName="m/z" unitCvRef="MS" />
-						</isolationWindow>
 						<selectedIonList count="1">
```

Comment:
@hendrikweisser is this a good idea to not write this out any more?

Reply:
Yes. The element actually isn't present in the input file that gets merged, so the new behaviour is correct.